analyzer/ldap/README.md

```diff
@@ -42,6 +42,8 @@ Here's what it has:
       - `base_object` (vector of 0..n search base objects specified)
       - `result_count` (number of result entries returned)
       - `result` (set of 1..n results from this uid+message_id)
+      - `filter` (search filter string)
+      - `attributes` (vector of 0..n "attributes", the attributes that were returned)
       - `diagnostic_message` (vector of 0..n diagnostic message strings)
```

Comment:
This is now neither in the order these fields appear now alphabetic order. Let's sort by the order they appear.
```suggestion
      - `diagnostic_message` (vector of 0..n diagnostic message strings)
      - `filter` (search filter string)
      - `attributes` (vector of 0..n "attributes", the attributes that were returned)
```